test/integration/datatype/test-integer.js

```diff
@@ -121,20 +121,20 @@ describe('integer with big value', () => {
       })
       .then((rows) => {
         assert.strictEqual(rows.length, 6);
-        assert.strictEqual(rows[0].v, -9007199254740991n);
-        assert.strictEqual(rows[1].v, 127n);
-        assert.strictEqual(rows[2].v, 128n);
-        assert.strictEqual(rows[3].v, 9007199254740991n);
-        assert.strictEqual(rows[4].v, 9007199254740992n);
-        assert.strictEqual(rows[5].v, 9007199254740993n);
+        assert.strictEqual(rows[0].v, BigInt(-9007199254740991));
+        assert.strictEqual(rows[1].v, BigInt(127));
+        assert.strictEqual(rows[2].v, BigInt(128));
+        assert.strictEqual(rows[3].v, BigInt(9007199254740991));
+        assert.strictEqual(rows[4].v, BigInt(9007199254740992));
```

Comment:
BigInt(9007199254740992) is unpredicable  that has to be replaced by string -> BigInt('9007199254740992')